src/Microsoft.VisualStudio.Editor.Razor/DefaultVisualStudioRazorParser.cs

```diff
@@ -3,7 +3,6 @@
 
```

Comment:
Do we want the entire parser to be torn down/re-created every time tag helpers change?

Reply:
Same answer as https://github.com/aspnet/Razor/pull/1759/files#r150061201.